docs/pages/10_view.adoc

```diff
@@ -0,0 +1,629 @@
+
+The following chapters will be easier to understand if you first grasp the concepts presented in the xref:07_transfer.adoc[Transfer model] chapter. There you can learn about transfer objects, their fields, relations and actions. These elements form the basis for building more sophisticated structures such as views and rows. Once you have a solid understanding of how transfer objects work, you can delve into the following chapters, where you can learn how to create a visual representation of data through views.
+
+== View
+
+A view is a transfer object that defines the visual presentation of data. It serves as a bridge between the raw data stored in entities and the user interface, providing a customizable and user-friendly representation.
+
+*Syntax:*
+
+[subs="quotes"]
+----
+[purple]#**view**# &lt;name> &#x5B;[purple]##**(**##&lt;entity> &lt;mapping-field>[purple]##**)**##]
+    &#x5B;[purple]##**dialog:**##([purple]##**true**##|[purple]##**false**##)]
+&#x5B;[purple]##**{**##
+    &#x5B;member] ...
+[purple]##**}**##][purple]#**;**#
+----
+
+or
+
+[subs="quotes"]
+----
+[purple]#**view**# &lt;name> &#x5B;[purple]##**maps**## &lt;entity> [purple]#**as**# &lt;mapping-field>]
+&#x5B;[purple]##**{**##
+    &#x5B;member] ...
+[purple]##**}**##][purple]#**;**#
+----
+
+The view syntax is used to define a visual representation of data. It begins with the keyword `view` followed by the chosen <name> for the view. Optionally, you can specify the entity it is associated with using (entity mapping-field).
+
+Inside the curly braces `{` and `}`, you can specify various components or members of the view, such as fields, actions, or other elements that contribute to its functionality. These components collectively define the visual interface and behavior of the view.
+
+The `dialog` modifier is used in the context of defining a view to specify whether the view should be displayed as a pop-up dialog window. When set to true, it indicates that the view is intended to be presented as a dialog, typically for tasks that require user input or confirmation before proceeding.
+
+*Example:*
+
+[source,jsl]
+----
+view CartView maps Customer as customer {
+	table OrderItemRow[] items <= customer.cart.orderItems;
+	field Integer price <= customer.cart.price label:"Total price ($):";
+	action ThankYouView order() label:"Order";
+}
+----
+
+In this example, a view named `CartView` is defined, and it is mapped to the `Customer` entity through the mapping field of `customer`. The view is designed to represent a shopping cart. Let's break down the components of this view:
+
+* *Table Component* (OrderItemRow[] items): The items table represents a collection of OrderItemRow rows. It is derived from the orderItems collection within the customer's cart.
```

Comment:
What will be the create form in case of a table instance creation? If tables are relations pointing to rows, and rows can only contain columns?

Reply:
https://blackbelt.atlassian.net/browse/JNG-5640